src/session/mod.rs

```diff
@@ -40,6 +44,9 @@ mod imp {
         pub chat_list: OnceCell<ChatList>,
         pub user_list: OnceCell<UserList>,
         pub selected_chat: RefCell<Option<Chat>>,
+        pub private_chats_notification_settings: RefCell<BoxedScopeNotificationSettings>,
+        pub group_chats_notification_settings: RefCell<BoxedScopeNotificationSettings>,
+        pub channel_chats_notification_settings: RefCell<BoxedScopeNotificationSettings>,
```

Comment:
@melix99 Would you like these fields in separate struct?

Reply:
No, this seems reasonable. What I'm not too much of a fan is having these Boxed types around, but I don't see a better solution now (other than modifying tdgrand to make all types derive glib::GBoxed, which I'm evaluating for a future update).